server/routes/api.js

```diff
@@ -115,4 +115,37 @@ routes.get('/pairing/paired', (req, res) => {
   });
 });
 
+routes.get('/wishlist/:familyId/read', (req, res) => {
```

Comment:
The route slug should read `'/wishlist/:familyId'` only.

Reply:
good catch.. totally slipped me up